packages/neo-one-cli/src/interactive/createCRUD.js

```diff
@@ -149,6 +149,24 @@ const renderTasks = (
   return output.join('\n');
 };
 
+const promptDelete = ({
+  cli,
+  crud,
+  argname,
+}: {|
+  cli: InteractiveCLI,
+  crud: CRUDResource<*, *>,
+  argname: string,
+|}) => {
+  const name = `${crud.resourceType.name} ${argname}`;
```

Comment:
nit: I'd just put this directly inline with `message`, no real reason to have it separate.